sig-vmware/charter.md

```diff
@@ -0,0 +1,60 @@
+# SIG VMware Charter
+
+This charter adheres to the conventions described in the [Kubernetes Charter README] and uses
+the Roles and Organization Management outlined in [sig-governance].
+
+## Scope
+
+The VMware SIG maintains and evolves the ability to run Kubernetes on VMware infrastructure.
+
+In addition to the cloud provider for the vSphere hypervisor, the SIG’s mission includes:
+
+- NSX and NSX-T, infrastructure components that provide software based networking service, with automated micro-segmentation at application and control plane levels - along with security, monitoring, logging, and features for multi-cluster, multi-site, and cross cloud integration. NSX-T is not vSphere specific, and is supported on other cloud providers (e.g. OpenStack). The scope of NSX and NSX-T interaction with Kubernetes goes beyond the CNI plug-in abstraction.
+- Desktop hypervisors (Fusion for OS X and Workstation for Linux and Windows), as supported by Minikube.
+- vCloud Director, which enables service providers to provision and manage multi tenant clouds, which can host Kubernetes.
+
+
+### In scope
+- Deploying Kubernetes of any type (standard or a packaged distribution from any vendor) on vSphere infrastructure.
+- Utilizing networking and storage features of VMware infrastructure through Kubernetes.
+- Determining and documenting best practices for configuring Kubernetes when running on vSphere infrastructure.
+- Determining and documenting best practices for configuring VMware infrastructure when supporting Kubernetes.
+- Determining and documenting best practices for using common configuration management and orchestration tools for deploying and managing Kubernetes on VMware infrastructure.
+- Discussing bugs and feature requests recorded as Kubernetes or VMware cloud provider issues on GitHub. These issues should be tagged with ``sig/vmware``.
+
+The authoritative source for SIG information is the [sigs.yaml] file.

```

Comment:
can you link here instead? https://github.com/kubernetes/community/tree/master/sig-vmware#subprojects